config/config.go

```diff
@@ -217,7 +216,7 @@ type ScrapeConfig struct {
 	// Indicator whether the scraped metrics should remain unmodified.
 	HonorLabels bool `yaml:"honor_labels,omitempty"`
 	// A set of query parameters with which the target is scraped.
-	Params url.Values `yaml:"params,omitempty"`
+	Params map[string]string `yaml:"params,omitempty"`
```

Comment:
> Cannot really check right now, but are there no characters that cannot be percent-encoded that could serve as a separator?

The value could be anything.

> Let's think this through or explicitly be open to do something breaking in the future

This change is non-breaking now, and we can do `__param2`, `__param3` etc. if anyone wants to touch more than one param in the future (which I can't think of a use case for at the moment).